File: iPad/VerBank-CSTSv3-IOSClientApi/ClientStation/api/exception/APIException.m
//
//  APIException.m
//  JEDIv7-CSTSv3-ClientAPI
//
//  Created by baolin on 13/5/24.
//  Copyright (c) 2013年 wangyubo. All rights reserved.
//

#import "APIException.h"

@implementation APIException


-(id)init:(NSString *)errCode errMessage:(NSString *)errMessage{
    if (self=[super init]) {
        _errCode=errCode;
        _errMessage=errMessage;
    }
    return self;
}

-(NSString *)getErrCode{
    return _errCode;
}

-(void)setErrCode:(NSString *)errCode{
    _errCode=errCode;
}

-(NSString *)getErrMessage
{
    return _errMessage;
}

-(void)setErrMessage:(NSString *)errMessage{
    _errMessage=errMessage;
}

@end

File: WeChatService/EX02-CSTSv3-ClientAPI/src/jedi/ex01/client/station/api/exception/APIException.java
package jedi.ex01.client.station.api.exception;

public class APIException extends Exception {
	
	private static final long serialVersionUID = 6803021127765807899L;
	
	private String errCode;
	private String errMessage;
	public APIException(String errCode,String errMessage){
		super();
		this.errCode=errCode;
		this.errMessage=errMessage;
	}
	public String getErrCode() {
		return errCode;
	}
	public void setErrCode(String errCode) {
		this.errCode = errCode;
	}
	public String getErrMessage() {
		return errMessage;
	}
	public void setErrMessage(String errMessage) {
		this.errMessage = errMessage;
	}
}
